crates/holochain_zome_types/src/info.rs

```diff
@@ -0,0 +1,39 @@
+use crate::header::ZomeId;
+use crate::zome::ZomeName;
+use holo_hash::AgentPubKey;
+use holo_hash::DnaHash;
+use holochain_serialized_bytes::prelude::*;
+
+/// The properties of the current dna/zome being called.
+#[allow(missing_docs)]
+#[derive(Clone, Debug, Serialize, Deserialize, SerializedBytes, PartialEq)]
+pub struct ZomeInfo {
+    pub dna_name: String,
+    pub dna_hash: DnaHash,
+    pub zome_name: ZomeName,
+    /// The position of this zome in the `dna.json`
+    pub zome_id: ZomeId,
+    pub properties: SerializedBytes,
+}
+
+/// The struct containing all information about the executing agent's identity.
+#[allow(missing_docs)]
+#[derive(Clone, Debug, Serialize, Deserialize, SerializedBytes, PartialEq)]
+pub struct AgentInfo {
+    /// The current agent's pubkey at genesis.
+    /// Always found at index 2 in the source chain.
+    pub agent_initial_pubkey: AgentPubKey,
+    /// The current agent's current pubkey.
+    /// Same as the initial pubkey if it has never been changed.
+    /// The agent can revoke an old key and replace it with a new one, the latest appears here.
+    pub agent_latest_pubkey: AgentPubKey,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct BundleInfo;
```

Comment:
```suggestion
pub struct AppInfo;
```

Again this will probably change, but no need to now.

Reply:
sure